src/client.js

```diff
@@ -56,7 +56,7 @@ const Client = function Client(config = {}) {
   };
 
   const ctxe = _.pick((this.configuration() || {}), ["organization", "id"]);
-  const logFactory = level => (message, ...data) => logger[level](message, { context: ctxe, data });
+  const logFactory = level => (message, data) => logger[level](message, { context: ctxe, data });
```

Comment:
It is problematic to unwrap this object when it is later sent and stored in the logging backend. we will now enforce receiving a single object here. No need for a major version bump since it's now consistent with the documented signature.